twiggy/tests/all/expectations/dominators_csv_does_not_summarize_garbage_if_all_items_are_reachable

```diff
@@ -0,0 +1,18 @@
+Id,Name,ShallowSize,ShallowSizePercent,RetainedSize,RetainedSizePercent,ImmediateDominator
```

Comment:
This test exists to show that we won't show a row summarizing unreachable items if there are no unreachable items